com.unity.render-pipelines.universal/Runtime/ForwardRenderer.cs

```diff
@@ -396,7 +394,9 @@ public override void Setup(ScriptableRenderContext context, ref RenderingData re
             }
 
 #if ENABLE_VR && ENABLE_XR_MODULE
-            if (cameraData.xr.hasValidOcclusionMesh)
+            // URP uses writes to depth buffer for the XR occlusion mesh pass.
+            // When copy depth is required, we turn off occlusion pass so that depth buffer contains valid data
```

Comment:
Why is this an issue? can you explain a bit more ?

Reply:
Added more comments in 586eabb

The issue is that occlusion mesh pass stores near clip plane value to the ooclusion mesh area in depth buffer.
When depth sharing is enabled, the depth buffer is used in timewarp for reprojection purpose and the near clip plane value in occlusion mesh region causes issue described in https://fogbugz.unity3d.com/f/cases/1242142/

Reply:
As we discusssed, reverted the code at: bd360b6
Forwarded the bug to WMR provider to be handled in provider layer.